app/attributes/attributecontroller.cpp

```diff
@@ -457,6 +457,8 @@ void AttributeController::updateOnFeatureChange()
         if ( shouldUseRememberedValue )
         {
           mFeatureLayerPair.featureRef().setAttribute( fieldIndex, rememberedValue );
+          itemData->setRawValue( rememberedValue );
+          emit formDataChanged( itemData->id(), { AttributeFormModel::RawValue } );
```

Comment:
It should (hopefully) not be necessary to emit the signal here as later we call the `derived`